### DISCUSSION TITLE
impossible to work with GPU

### DISCUSSION THREAD

**Discussion by Sequential-circuits** [nextcloud/recognize#1069] [Q&A]
it is impossible to make it work with GPU

It insists in saying Classifier process output: 2024-01-04 16:18:12.129427: W tensorflow/stream_executor/platform/default/dso_loader.cc:64] Could not load dynamic library 'libcusolver.so.11'; dlerror: libcusolver.so.11: cannot open shared object file: No such file or directory

I have cuda 11 and cuDNN SDK 8.1.0 installed and LD_LIBRARY set correctly to point to libcusolver.so.11

And, finally, even if I uninstall tensorflow completely, it still shows that same error message

**Comment by marcelklehr** [comment]
Hi, thank you for your new attempt to formulate this :blue_heart: 
Could you please file an issue for this and fill out the issue template, so we can help you better? That way we get a lot of useful information about your setup without having to ask our way through :)
